### DISCUSSION TITLE
DocTR PyTorch: CPU-Only Version

### DISCUSSION THREAD

**Discussion by jayanthsaish** [mindee/doctr#1728] [General]
I am encountering an issue in production where the doctr package includes CUDA dependencies, which significantly increase its size. However, I only need the CPU version. Could you please advise where I can find a CPU-only version?

Currently, I am installing it using **pip install python-doctr**, I have already downloaded Torch separately.

**Comment by felixdittrich92** [comment]
Hi @jayanthsaish :wave:,

You have installed the cpuonly version before:

pip:
```
pip3 install torch torchvision --index-url https://download.pytorch.org/whl/cpu
```

conda:
```
conda install pytorch torchvision cpuonly -c pytorch
```

?

For prod systems you can also take a look here:
https://github.com/felixdittrich92/OnnxTR  --> no PyTorch or TensorFlow required / much faster / more lightweight

**Reply by felixdittrich92** [reply]
real latency benchmarks:
https://github.com/felixdittrich92/OnnxTR?tab=readme-ov-file#benchmarks

**Reply by jayanthsaish** [reply]
Okay thank you : )
